src/components/MapComp/index.tsx

```diff
@@ -22,6 +23,25 @@ function MapComp() {
     mapRef.current = new mapboxgl.Map(mapConfig);
     const map = mapRef.current;
 
+    const getFeaturesOnScreen = () => {
+      const mapboxFeaturesOnScreen = map.querySourceFeatures('placeList');
+
+      const uniqueIds = new Set<number>();
+
+      mapboxFeaturesOnScreen.forEach((feature) => {
+        const id = feature.properties?.id;
+        if (!uniqueIds.has(id)) {
+          uniqueIds.add(id);
+        }
+      });
+
+      setFeaturesOnScreen(features.filter((feature: CustomGeoJSONFeatures) => uniqueIds.has(feature.properties.id)));
+    };
+
+    const initFeaturesOnScreen = () => {
+      getFeaturesOnScreen();
+    };
```

Comment:
위처럼 get함수를 배열 리턴값으로 정했다면
```suggestion
    const initFeaturesOnScreen = () => {
      const featuresOnScreen = getFeaturesOnScreen();
     setFeaturesOnScreen(featuresOnScrren);
    };
```
이런식으로 작성할 수 있을 것 같아요, init뿐만 아니라 다른데에도 쓰이기 때문에 함수 네이밍도 바꾸어 주어야 할것 같아요. updateFeatureScreen 아니면 음.. 더 좋은게 있을까요?

Reply:
getFeaturesOnScreen 함수 배열 리턴으로 변경하고, init을 update로 네이밍 변경하겠습니다. 감사합니다!!

Reply:
init 함수가 단순히 get함수만 실행중이어서 삭제하고 get을 update로 네이밍 변경해보겠습니다